### PR TITLE
Polish root README.md

### PR DESCRIPTION
**Pull request by blorticus**
On the root README.md:
  - normalized markdown layout
  - corrected small grammar and spelling errors
  - standardized idiomatic English
  - removed some reference redundancies

### CODE REVIEW

**Comment by free5gc-org** [comment]
Hi @blorticus ,

Because we have release our v3.0.4 version. Please help us fix the conflict and our check on it.

Thanks a lot.

File: README.md [hunk 1]
```diff
@@ -22,268 +22,320 @@
...
+```bash
+sudo sysctl -w net.ipv4.ip_forward=1
+sudo iptables -t nat -A POSTROUTING -o <dn_interface> -j MASQUERADE
+sudo systemctl stop ufw
+```
+
+### B. Install Control Plane Elements
+    
+1. Clone the free5GC repository
+    * To install the latest stable build (v3.0.3):
```
Comment (free5gc-org) [inline README.md:143]: Here should be change to v3.0.4

File: README.md [hunk 2]
```diff
@@ -22,268 +22,320 @@
...
+
     ```bash
-    ./build.sh
+        cd ~
+        git clone --recursive -b v3.0.3 -j `nproc` https://github.com/free5gc/free5gc.git
```
Comment (free5gc-org) [inline README.md:147]: Here should be change to v3.0.4

File: README.md [hunk 3]
```diff
@@ -22,268 +22,320 @@
...
+    * To do so individually (e.g., AMF only):
 
-    Get Linux kernel module 5G GTP-U
     ```bash
-    git clone -b v0.1.0 https://github.com/PrinzOwO/gtp5g.git
-    cd gtp5g
-    make
-    sudo make install
+        cd ~/free5gc
+        go build -o bin/amf -x src/amf/amf.go
```
Comment (free5gc-org) [inline README.md:175]: We have update build procedure to `make amf`, `make smf`, `make all`, ...

File: README.md [hunk 4]
```diff
@@ -22,268 +22,320 @@
...
+1. As noted above, the GTP kernel module used by the UPF requires that you use Linux kernel version `5.0.0-23-generic`.  To verify your version:
+
+```bash
+uname -r
+```
+
+2. Retrieve the 5G GTP-U kernel module using `git` and build it
+
+```bash
+git clone -b v0.1.0 https://github.com/PrinzOwO/gtp5g.git
```
Comment (free5gc-org) [inline README.md:196]: Here must be changed to v0.2.0

File: README.md [hunk 5]
```diff
@@ -22,268 +22,320 @@
...
+cd gtp5g
+make
+sudo make install
+```
+
+3. Build the UPF
+
```
Comment (free5gc-org) [inline README.md:203]: Auto build from `make upf` under free5gc folder.

Manual build by command below.

**Review by free5gc-org — COMMENTED** [review]
Need to update the README to fit v3.0.4 version.

**Comment by blorticus** [comment]
Done.  The branch in the pull request was re-synchronized with the upstream/master, then the changes re-applied against the newer README.md.

File: README.md (hunk 3 above)
Comment (free5gc-org) [reply README.md:178]: The building process still remain as the old way. It need to be using make to build it now.

**Comment by free5gc-org** [comment]
The building process still remain as the old way. It need to be using make to build it now.

File: README.md (hunk 1 above)
Comment (blorticus) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (blorticus) [reply README.md]: Done.

File: README.md (hunk 4 above)
Comment (blorticus) [reply README.md]: Done.

File: README.md (hunk 5 above)
Comment (blorticus) [reply README.md]: Done.

**Comment by blorticus** [comment]
Indeed.  My apologies.  I appear to have missed that one.  It is now fixed, as well.

**Review by free5gc-org — APPROVED** [review]
LGTM

**Pull request merged**